src/Configuration.cpp

```diff
@@ -1,10 +1,10 @@
 #include "Configuration.hpp"
 
-Configuration::Configuration() : _parsed(false), _port(), _password()
+Configuration::Configuration() : _network_name("ft_copains"), _server_name("localhost"), _server_version(SERVER_VERSION), _server_creation_date("00:00:00 Jan 01 1970"), _user_modes(""), _channel_modes("iko"), _parsed(false), _port(), _password()
```

Comment:
I'd rather use a real date than 00:00:00 Jan 01 1970

Reply:
No thank you